internal/schema/stacks/1.8/deployment_block.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/terraform-schema/internal/schema/refscope"
+	"github.com/hashicorp/terraform-schema/internal/schema/tokmod"
+)
+
+func deploymentBlockSchema() *schema.BlockSchema {
+	/*
+		Reference: https://github.com/hashicorp/tfc-agent/blob/main/core/components/stacks/tfdeploycfg/deployment.go
+		TODO:
+			- Source better descriptions
+			- Verify all attributes are added here
+	*/
```

Comment:
These comments were useful for me to keep things straight while building this out, but maybe not so useful moving forward.